src/core/less/global/theme-extras.less

```diff
@@ -0,0 +1,94 @@
+// Additional theme classes or attributes to be applied in JSON for more custom style 
```

Comment:
this whole file needs to be deleted, this is for the demo course on git

that is unless we can make arguments to keep .divider-block, .assessment .graphic-component and .component-background?